### DISCUSSION TITLE
"What Happens If an Exception Isn’t Handled?"

### DISCUSSION THREAD

**Discussion by ShervanN** [ProfessionalCSharp/ProfessionalCSharp2021#114] [Q&A] [answered]
Hi,

Could you please write a complete example for "What Happens If an Exception Isn’t Handled?" ? I didn't understand well.
Page: 278 - CHAPTER 10 Errors and ExcEptions

Thank you

**Comment by christiannagel** [comment, marked as answer]
If an exception is not handled, the runtime gets a chance to deal with this. What happens is implementation defined.

With a console application you'll see the application to stop and to show the exception.
With background services, there's a breaking change with .NET 6 to also stop the application - more information on this:

[Unhandled exceptions from a BackgroundService](https://docs.microsoft.com/dotnet/core/compatibility/core-libraries/6.0/hosting-exception-handling)

Does this help?
